Diary/Entity/PersistentContainer.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  PersistentContainer.swift
+//  Diary
+//
+//  Created by MINT, BMO on 2023/09/04.
+//
+
+import CoreData
+
+class PersistentContainer: NSPersistentContainer {
+    func saveContext(backgroundContext: NSManagedObjectContext? = nil) {
+        let context = backgroundContext ?? viewContext
+        guard context.hasChanges else { return }
```

Comment:
[save()](https://developer.apple.com/documentation/coredata/nsmanagedobjectcontext/1506866-save) 공식문서에 보면 
`Always verify that the context has uncommitted changes (using the [hasChanges](https://developer.apple.com/documentation/coredata/nsmanagedobjectcontext/1506954-haschanges) property) before invoking the save: method. Otherwise, Core Data may perform unnecessary work.`
라는 말이 있습니다. 때문에 save 이전에 hasChanges를 이용해 체크해 주었습니다.